dotnet/build-packages.sh

```diff
@@ -1,5 +1,5 @@
 RELEASE_VERSION="2.43.000"
```

Comment:
I'm _fairly_ sure this should have been bumped to 2.43.001 though? 😅 
 
```suggestion
RELEASE_VERSION="2.43.001"
```